soccer/gameplay/evaluation/position.py

```diff
@@ -0,0 +1,48 @@
+import main
+import robocup
+import constants
+import math
+
+'''calculates the distance of a robot to the ball
+def distance(robot):
+	x = abs(main.ball().pos.x - robot.pos.x)
+	y = abs(main.ball().pos.y - robot.pos.y)
+	dist = math.sqrt((x*x) + (y*y))
+	return dist
+'''
+	
+'''returns all robots, as well as their average positions, that are within a given radius of the ball'''
+def ball_vicinity(us, them, radius):
```

Comment:
Since us and them are always going to be in main.our_robots() and main.their_robots() you don't need to have them be passed in and can just directly access them in the eval function itself.